marklogic-data-hub/src/main/java/com/marklogic/hub/dataservices/EntitySearchService.java

```diff
@@ -211,10 +210,11 @@ private void deleteSavedQuery(BaseProxy.DBFunctionRequest request, String id) {
    * @param schemaName	provides input
    * @param viewName	provides input
    * @param limit	provides input
+   * @param sortOrder	provides input
    * @param columns	provides input
    * @return	as output
    */
-    com.fasterxml.jackson.databind.JsonNode getOpticPlan(String structuredQuery, String searchText, String queryOptions, String schemaName, String viewName, Long limit, Stream<String> columns);
+    com.fasterxml.jackson.databind.JsonNode getOpticPlan(String structuredQuery, String searchText, String queryOptions, String schemaName, String viewName, Long limit, com.fasterxml.jackson.databind.node.ArrayNode sortOrder, Stream<String> columns);
```

Comment:
Just a note for @ehennum - an improvement to DS would be the ability to define the equivalent of "structured types" for DS interfaces so that we don't have long lists of parameters, but rather have a generated class with multiple properties.